docs/how-to-configure-new-repository.md

```diff
@@ -30,13 +30,10 @@ for the repository.
    PRs. See the [Policies](#policies) section that suggest to ensure that `Require
    review from Code Owners` is checked.
 4. The team `foo-approvers` has `Write` permissions for the repository.
-5. The team `foo-maintainers` has `Maintain` permissions for the
+5. The team `foo-maintainers` has `Admin` permissions for the
    repository.
```

Comment:
How about this?

```suggestion
5. The team `foo-maintainers` has `Maintain` permissions for the
   repository.
   In individual cases, for example, if this is required to allow maintainers to publish releases independently,
   the team `foo-maintainers` can be granted `Admin` permissions instead.
```
This is similar to the previous wording but would reflect the status quo where some maintainer _teams_ rather than individuals have admin access and would thus fix the current state for now without making `Admin` the default for _all_ (new) repos.

If solutions can be found that suit all maintainers (like the suggestion provided by @trask in https://github.com/open-telemetry/community/pull/983#issuecomment-1049155557), we can remove this clause again in future.

Reply:
@tigrannajaryan does this work for you?

Reply:
Yes, works for me.